scripts/scrutinizer-stub.js

```diff
@@ -16,23 +16,24 @@
 
 const fs = require('fs')
 const yaml = require('js-yaml')
-const markdown = require('markdown').markdown
+const md2jsonml = require('md2jsonml')
 const _ = require('lodash')
 const path = require('path')
 const PROJECT_DIRECTORY = path.resolve(__dirname, '..')
 
 const getHighlights = (readme) => {
-  const tree = _.tail(markdown.parse(readme))
+  const tree = _.tail(md2jsonml(readme))
   return tree[3].slice(1).map((highlight) => {
+    const highlightPair = highlight.slice(1);
     return {
-      title: highlight.slice(1)[0][1],
-      description: highlight.slice(1)[1].replace(/^:\s+/, '')
+      title: _.get(highlightPair, [0, 1], ""),
+      description: _.get(highlightPair, [1], "").replace(/^:\s+/, '')
```

Comment:
The previous `highlight.slice(1)[1].replace(/^:\s+/, '')` line now breaks. Need to understand how the parsing changed after introducing md2jsonml.